### PR TITLE
:sparkles: Add documents for bot types and medallia

### PR DESCRIPTION
**Pull request by ahuppert**
Work in Progress - do not merge

### CODE REVIEW

File: pages/Documents/ConversationalAI/Third-PartyBots/bot-conversation-errors.md [hunk 1]
```diff
@@ -480,4 +480,27 @@ This indicates an issue fetching the SDEs.
...
+#### Reasons
+Most of the time this is caused by wrong credentials or because the bot instance on vendor side is not running.
+#### Solutions
+Check if your bot credentials are configured correctly in Third-Party Bots and the bot is healthy on the vendor side.
+
+
+### com.liveperson.bot-connectors-worker.error.aivendor.medallia.parse-message.parser-error
+
+#### Description
+Parsing the response of the Madallia Adapter failed.
```
Comment (snikic) [inline pages/Documents/ConversationalAI/Third-PartyBots/bot-conversation-errors.md:499]:
```suggestion
Parsing the response of the Medallia Adapter failed.
```

File: pages/Documents/ConversationalAI/Third-PartyBots/ibm-watson-assistant-version-2.md [hunk 2]
```diff
@@ -50,24 +50,24 @@ You need to fill in the following infromation:
   </thead>
   <tbody>
   <tr>
-    <td>Workspace URL</td>
-    <td>Watson Assistant Workspace URL. Note that this workspace URL changes according to your account. In order to figure out which URL is correct for you, check the location of your Assistant in IBM's Cloud console. Then, choose the corresponding URL based on geo-location. For example, if your IBM location is "eu-gb", the London URL is the correct one for you.</td>
-    <td>https://api.eu-de.assistant.watson.cloud.ibm.com</td>
+    <td>Client ID</td>
```
Comment (snikic) [inline pages/Documents/ConversationalAI/Third-PartyBots/ibm-watson-assistant-version-2.md:53]: Sure that his should all go into ibm-watson-assistant-version-2.md ?

File: yarn.lock [hunk 3]
```diff
@@ -0,0 +1,3549 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
+# yarn lockfile v1
```
Comment (snikic) [inline yarn.lock]: did you add this file on purpose?

**Review by snikic — CHANGES_REQUESTED** [review]

File: yarn.lock (hunk 3 above)
Comment (ahuppert) [reply yarn.lock]: no. apparently yarn was never used before
will remove it

File: pages/Documents/ConversationalAI/Third-PartyBots/bot-conversation-errors.md [hunk 4]
```diff
@@ -480,4 +480,27 @@ This indicates an issue fetching the SDEs.
...
+#### Reasons
+In this case the Medallia Adapter sent some data we did not expect.
+#### Solutions
+Contact the Medallia Adapter and provide them the data of the conversation error.
```
[the remark this answers is not in the document]
Comment (ahuppert) [reply pages/Documents/ConversationalAI/Third-PartyBots/bot-conversation-errors.md:503]: ok

File: pages/Documents/ConversationalAI/Third-PartyBots/ibm-watson-assistant-version-2.md (hunk 2 above)
Comment (ahuppert) [reply pages/Documents/ConversationalAI/Third-PartyBots/ibm-watson-assistant-version-2.md]: no :) will fix it

File: pages/Documents/ConversationalAI/Third-PartyBots/bot-conversation-errors.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (ahuppert) [reply pages/Documents/ConversationalAI/Third-PartyBots/bot-conversation-errors.md]: changed it now to support for now until we have proper solution path

**Pull request merged**